cub/cmake/CubHeaderTesting.cmake

```diff
@@ -41,11 +41,12 @@ function(cub_add_header_test label definitions)
 endfunction()
 
 # Wrap Thrust/CUB in a custom namespace to check proper use of ns macros:
-set(header_definitions 
-  "THRUST_WRAPPED_NAMESPACE=wrapped_thrust" 
+set(header_definitions
+  "THRUST_WRAPPED_NAMESPACE=wrapped_thrust"
   "CUB_WRAPPED_NAMESPACE=wrapped_cub")
 cub_add_header_test(base "${header_definitions}")
 
 list(APPEND header_definitions "CUB_DISABLE_BF16_SUPPORT")
+list(APPEND header_definitions "_CCCL_NO_SYSTEM_HEADER")
```

Comment:
Why is this only set for the bf16 header tests, but not the base header tests?

Reply:
🤦